docs/setup/upload.md

```diff
@@ -0,0 +1,24 @@
+# Uploading HULK
+
+This section assumes, you have set up your development environment, and flashed a Yocto image to the NAO.
```

Comment:
I think this sentence needs neither comma.
```suggestion
This section assumes that you have set up your development environment and flashed a Yocto image to the NAO.
```

Reply:
done